src/index.js

```diff
@@ -21,8 +21,11 @@ import createGeoRasterLayer from "./utils/create-georaster-layer.js";
 // that item is missing a type
 
 const isImageType = type => MIME_TYPES.BROWSER.includes(type);
-const isAssetCOG = asset =>
-  MIME_TYPES.COG.includes(asset.type) && typeof asset.href === "string" && asset.href.length > 0;
+const isAssetCOG = asset => isAssetGeoTiff(asset, true);
+const isAssetGeoTiff = (asset, cloutOptimized = false) => {
```

Comment:
little typo here and on the next line: `cloutOptimized` -> `cloudOptimized`

Reply:
Thanks, fixed.